deno-runtime/main.ts

```diff
@@ -54,13 +53,19 @@ function wrapAppCode(code: string): (require: (module: string) => unknown) => Pr
     ) as (require: (module: string) => unknown) => Promise<Record<string, unknown>>;
 }
 
-async function handlInitializeApp({ id, source }: { id: string; source: string }): Promise<void> {
-    source = sanitizeDeprecatedUsage(source);
+async function handlInitializeApp(appPackage: IParseAppPackageResult): Promise<void> {
+    const source = sanitizeDeprecatedUsage(appPackage.files[appPackage.info.classFile]);
+
     const require = buildRequire();
     const exports = await wrapAppCode(source)(require);
+
     // This is the same naive logic we've been using in the App Compiler
-    const appClass = Object.values(exports)[0] as typeof App;
-    const app = new appClass({ author: {} }, console, AppAccessorsInstance.getDefaultAppAccessors());
+    // Applying the correct type here is quite difficult because of the dynamic nature of the code
+    // deno-lint-ignore no-explicit-any
+    const appClass = Object.values(exports)[0] as any;
+
+    // What do I do here? D:
```

Comment:
```suggestion
    // What do I do here? D:
```

Cast to `any`, that's what we can do here